tests/test_unet.py

```diff
@@ -141,7 +141,7 @@ def test_script_without_running_stats(self):
             num_res_units=0,
             norm=("batch", {"track_running_stats": False}),
         )
-        test_data = torch.randn(16, 1, 16, 8)
+        test_data = torch.randn(16, 1, 16, 4)
```

Comment:
should we add an unittest for wrong input case?

Reply:
I didn't test it because the error will be raised from PyTorch APIs instead of MONAI code.
Anyway, let me add a unit test for it.

Thanks.

Reply:
Hi @yiheng-wang-nv ,

Added unit test in the latest commit.

Thanks.